.github/workflows/test-mingw.yml

```diff
@@ -93,14 +78,4 @@ jobs:
         with:
           file: ./coverage.xml
           flags: GHA_Windows
-          name: ${{ matrix.name }}
-
-  success:
-    permissions:
-      contents: none
-    needs: build
-    runs-on: ubuntu-latest
-    name: MinGW Test Successful
```

Comment:
We'll need to adjust https://github.com/python-pillow/Pillow/blob/0d54cf5c7fad29d7382ccb8927bb763b0dc4be7c/.github/mergify.yml#L10 to something like "Test MinGW"

Reply:
Sorry, I'm not following at all.

I initially thought the success job could be removed it there was only one job, but now I realise that I'd removed something mergify was using, and so I've restored it.

Still not sure what you're suggesting though.

Reply:
Sorry for not being clear :) 

Yeah, so mergify needs to depend on something. When we had a matrix here, mergify depended on the final noop, success job.

But now this workflow has a single job, mergify can depend on that single job, and we can avoid the overhead of spinning up the final success job. 

In other words, we can do the same as the lint workflow.

Reply:
Oh, I didn't realise you'd referenced mergify.yml

Thanks. I've pushed a different commit, see what think.

Reply:
Looks good 👍